NuGet.config

```diff
@@ -5,10 +5,7 @@
     <clear />
     <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
     <add key="darc-pub-microsoft-msbuild-e901037-2" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-microsoft-msbuild-e901037f-2/nuget/v3/index.json" />
-    <add key="darc-pub-dotnet-core-setup-65f04fb" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-core-setup-65f04fb6/nuget/v3/index.json" />
```

Comment:
@mmitche were some of these needed? Why are they getting removed?

Reply:
This is https://github.com/dotnet/arcade/issues/4642 - it's a bug related to identical SHAs existing in the internal & external branches